.github/workflows/contrib_checks.yml

```diff
@@ -287,7 +287,6 @@ jobs:
       - uses: prefix-dev/setup-pixi@v0.3.0
         with:
           pixi-version: v0.6.0
-          cache: true
```

Comment:
what did this do and why don't we want it anymore? :)

Reply:
It builds a ~600 MB cache that takes 4 minutes to save. I'm not really sure why, but everything is still fast woithout it.